walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
This change should be removed from this PR (it might help to have a separate branch off of master that you use for experimenting with the walkthroughs)

Reply:
i'm still seeing this change, you should just convert "getRandomQuoteUsingArrowFunctions" back to "getRandomQuote"

Reply:
Got it, commit did not see this change but I made sure to add this git file